lib/ansible/modules/system/syspatch.py

```diff
@@ -113,7 +113,8 @@ def syspatch_run(module):
     reboot_needed = False
     warnings = []
 
-    # Setup command flags
+    run_flag = []
```

Comment:
Defaulting `run_flag` to `[]` will cause `syspatch(8)` to apply all available patches. Since we don't set this again when `apply` is False, default it to `['-c']` so `syspatch(8)` will check for, rather than apply, available patches:

```suggestion
    # Set safe defaults for run_flag and check_flag
    run_flag = ['-c']
```